src/phoenix/server/app.py

```diff
@@ -153,12 +155,14 @@ def __init__(
         graphiql: bool = False,
         corpus: Optional[Model] = None,
         streaming_last_updated_at: Callable[[], Optional[datetime]] = lambda: None,
+        cache_for_dataloaders: Optional[CacheForDataLoaders] = None,
```

Comment:
Suggestion: How about just `dataloader_cache`?

Reply:
i want the `dataloaders` part to be plural, but `dataloaders_cache` doesn't have the same ring to it